### PR TITLE
Initial Product creation Docs

### PR DESCRIPTION
**Pull request by GabrielBlondeau-Kirby**
First step for Product docs.

- Create
- get_product_id

### CODE REVIEW

File: source/includes/_products.md [hunk 1]
```diff
@@ -1 +1,255 @@
...
+
+## Product Object
+
+Attribute | Type | Description
+--------- | ---- | -------
+active | `boolean` | Flag that sets Product object to active.
+created | `boolean` | The date the Product was generated.
+modified | `boolean` | The date the Product was last updated.
+deleted | `boolean` | Flag that set Product object to deleted.
+company_take |
```
Comment (pedropregueiro) [inline source/includes/_products.md]: take away things like this, that we don't know if we're using or not

File: source/includes/_products.md [hunk 2]
```diff
@@ -1 +1,255 @@
...
+name | `string` | The name of the product
+price | `float` | The price of the product
+currency |  `string` | Currency of the product (NOK, EUR, ...) ISO 4217
+business_rules | `list` |
```
Comment (pedropregueiro) [inline source/includes/_products.md]: take this away as well

File: source/includes/_products.md [hunk 3]
```diff
@@ -1 +1,255 @@
...
+description | `string` | Product description
+short_description | `string` | Short description about a product
+main_product | `boolean` | To know if a product is main product or not
+_sub_products | `list` | It is a list of sub products under the main product.
+parents | `list` | The sub product parent, overhead product.
+default_position | `geoPoint` | Define Product position.
+max_distance | `int` |
+tags | `list` | List of Tags associated with Product.
+properties | `dict` | Properties is a dictionary where you could add product properties.
+path | `string` | Path is the Product URL for website
```
Comment (pedropregueiro) [inline source/includes/_products.md]: take this away as well

File: source/includes/_products.md [hunk 4]
```diff
@@ -1 +1,255 @@
...
+X-Share-Api-Key: <shareactor-api-key>
+Host: api.shareactor.io
+
+{
+  "product": 
+  {
+        "description": "It is a test product",
+        "currency": "NOK",
+        "price": 3.14,
+        "name": "product_name",
```
Comment (pedropregueiro) [inline source/includes/_products.md]: use proper names "Product Name", otherwise people might think they need to use `_`

Comment (GabrielBlondeau-Kirby) [inline source/includes/_products.md]: OK, I keep in mind

**Comment by GabrielBlondeau-Kirby** [comment]
About attributes:
- slug
- max_distance

What is their description ?

**Comment by KristopherJordan** [comment]
Fix for the future:

slug
max_distance
What is their description ?

**Pull request merged**